source/logic/toasts.js

```diff
@@ -12,9 +12,16 @@ function setDB(database) {
 
 /** *Find the Secondings of specified seconder for the purposes of Crit Toast and Rewarded Toast tracking*
  * @param {string} seconderId
- */
-function findRecentSecondings(seconderId) {
-	return db.models.Seconding.findAll({ where: { seconderId, createdAt: { [Op.gt]: dateInPast({ d: 2 }) } } });
+ * @param {object} recency How far in the past to look for recent secondings. Defaults to 2 days.
```

Comment:
I think using the [Partial TypeScript utility type](https://www.typescriptlang.org/docs/handbook/utility-types.html#partialtype) would be more expressive in this case.